src/components/content/content.stories.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+import Content from './content.js';
+
+storiesOf('Content', module)
```

Comment:
Again we shouldn't see redundant prop information.

![image](https://user-images.githubusercontent.com/26925206/53880326-a42f7380-4008-11e9-9869-4f0c34693205.png)

Reply:
thanks, done